src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -1199,12 +1199,19 @@
       #(not (or (empty? %) (kitchensink/string-contains? "[" %)))
       (reverse containment-path)))))
 
-(def store-resources-column? (atom false))
-(defn maybe-resources
-  [row-map]
-  (if @store-resources-column?
+(def enterprise-data? (atom false))
```

Comment:
in the database? Not sure why we would. For what it's worth this is just a generalization of the atom that was already there; just wanted to change it not to be resource-specific.